src/cryoemservices/cli/clem_lif_to_stack.py

```diff
@@ -29,11 +30,21 @@ def run():
         default=1,
         help="Number of processes to run",
     )
-
     args = parser.parse_args()
 
-    convert_lif_to_stack(
-        file=Path(args.lif_path),
-        root_folder=args.root_dir,
-        number_of_processes=args.num_procs,
+    # Load args
+    lif_file = Path(args.lif_file)
+    root_folder: str = args.root_folder
+    num_procs: int = args.num_procs
```

Comment:
I don't think it helps much to convert these from args to variables as you aren't then doing anything with them